contracts/sidechain/CrossChainMessages.sol

```diff
@@ -8,7 +8,7 @@ pragma solidity 0.8.11;
  */
 library CrossChainMessages {
     /// @dev Magic Bytes to pad the custom message with
-    /// bytes4(keccak256("_isCustomMessage(bytes)"))
+    /// bytes4(keccak256("isCustomMessage(bytes)"))
```

Comment:
This should be reverted, `0x7a7f9946 == bytes4(keccak256("_isCustomMessage(bytes)"))`